### PR TITLE
docs: clarify config-file

### PR DESCRIPTION
**Pull request by alikhajeh1**

### CODE REVIEW

**Review by aliscott — APPROVED** [review]

File: docs/features/config_file.md [hunk 1]
```diff
@@ -23,38 +23,79 @@ Infracost configuration values are chosen in this order:
...
+  <td><code>name</code></td>
+  <td>Optional. String. Defaults to code path, workspace or Terraform/Terragrunt module within a repo. Name of project to use in all outputs (CLI, CI/CD integrations and Infracost Cloud).</td>
+</tr>
+<tr>
+  <td><code>usage_file</code></td>
+  <td>Optional. String. Path to Infracost usage file that specifies values for <a href="/docs/features/">usage_based_resources</a>. Relative to config-file path.</td>
+</tr>
+<tr>
+  <td><code>env</code></td>
+  <td>Optional. Map of strings. Environment variables that are passed to the project during processing. Also supports referencing existing environment variables using the syntax TODO. Environment variables that start with <code>INFRACOST_</code> are global in scope (not per-project) and cannot be used inside this parameter. For example:
```
Comment (alikhajeh1) [inline docs/features/config_file.md:60]: @aliscott any ideas on how I can change this and the next TODO to `<code>${MY_ENV_VAR}</code>`? 
When I try that, I get an error "MY_ENV_VAR is not defined".

File: docs/features/config_file.md [hunk 2]
```diff
@@ -23,38 +23,79 @@ Infracost configuration values are chosen in this order:
...
+  <td>Required. String. Path to the Terraform directory or JSON/plan file. The path is relative to the present working directory. A path can be repeated with different parameters, e.g. for multiple workspaces.</td>
+</tr>
+<tr>
+  <td><code>include_all_paths</code></td>
+  <td>Optional. Boolean. Defaults to false meaning that Infracost will autodetect only <b>root</b> Terraform modules. Setting this to true forces the autodetect function to estimate all directories (i.e. root and non-root modules) with valid project files, down to a max depth of 10 directories.</td>
+</tr>
+<tr>
+  <td><code>exclude_paths</code></td>
+  <td>Optional. Array of strings. Array of directory paths to exclude from evaluation, relative to <code>path</code> of project. Supports glob patterns too, for example:
+  <pre>
```
Comment (alikhajeh1) [inline docs/features/config_file.md:43]: @aliscott I couldn't figure-out how to style the `<pre>` blocks to break the lines and show the whitespaces correctly, the `<br/>` hack worked for new lines but updating `src/css/docs.css` didn't help. Feel free to push to this branch.

File: docs/features/config_file.md [hunk 3]
```diff
@@ -23,38 +23,79 @@ Infracost configuration values are chosen in this order:
...
+  <td>Optional. Map of strings. Input variables to use when parsing the Terraform HCL code, similar to Terraform's <code>-var</code> flag. For example:
+  <pre>
+  terraform_vars:<br/>
+    instance_count: 5<br/>
+    artifact_version: foobar
+  </pre></td>
+</tr>
+<tr>
+  <td><code>terraform_var_files</code></td>
+  <td>Optional. Array of string. Variable files to use when parsing Terraform HCL code, similar to Terraform's <code>-var-file</code> flag. For example:
```
Comment (alikhajeh1) [inline docs/features/config_file.md:78]: These file paths are relative to the `<code>path</code>` of the project?

File: docs/features/config_file.md [hunk 4]
```diff
@@ -23,38 +23,79 @@ Infracost configuration values are chosen in this order:
...
+    - app/*/ignore_dir
+  </pre></td>
+</tr>
+<tr>
+  <td><code>name</code></td>
+  <td>Optional. String. Defaults to code path, workspace or Terraform/Terragrunt module within a repo. Name of project to use in all outputs (CLI, CI/CD integrations and Infracost Cloud).</td>
+</tr>
+<tr>
+  <td><code>usage_file</code></td>
+  <td>Optional. String. Path to Infracost usage file that specifies values for <a href="/docs/features/">usage_based_resources</a>. Relative to config-file path.</td>
```
Comment (alikhajeh1) [inline docs/features/config_file.md:56]: > Relative to the config-file path

is this correct?

**Comment by alikhajeh1** [comment]
@aliscott can you please review this again? specifically the comments I've added to the PR

File: docs/features/config_file.md (hunk 3 above)
Comment (aliscott) [reply docs/features/config_file.md:78]: Yes

File: docs/features/config_file.md (hunk 2 above)
Comment (aliscott) [reply docs/features/config_file.md:43]: Use a React string:
```
<pre>
{`exclude_paths:
  - projects/myproject
  - test-*
  - app/*/ignore_dir`}
</pre>
```

File: docs/features/config_file.md (hunk 1 above)
Comment (aliscott) [reply docs/features/config_file.md:60]: You can try HTML entities:
```
$&#123;MY_ENV_VAR&#125;
```

File: docs/features/config_file.md (hunk 4 above)
Comment (aliscott) [reply docs/features/config_file.md:56]: No, it's relative to the working directory you run `infracost` from 🤦

**Pull request merged**